src/Money.php

```diff
@@ -426,6 +426,20 @@ public function allocateTo($n)
         return $this->allocate(array_fill(0, $n, 1));
     }
 
+    /**
+     * @param Money $money
+     *
+     * @return string
+     */
+    public function ratioOf(Money $money)
+    {
+        if (!$money->amount) {
```

Comment:
Shouldn't this be done in the `divide` methods itself cause it's a Division problem and no ratio one!?